test/no-vars-test.output.js

```diff
@@ -56,8 +56,7 @@ do {
 })();
 
 (() => {
-  let a = 1;
-  const b = 2, c = 3;
+  let a = 1, b = 2, c = 3;
```

Comment:
It is, in that I didn't implement splitting `var = 1, b = 2, c = 3;` apart, I just took the lowest common denominator.  On the other hand, it is still semantically correct and it's closer to what the user initially wrote.